jwst/datamodels/schemas/miri_ramp.schema.yaml

```diff
@@ -1,5 +1,6 @@
 allOf:
 - $ref: core.schema.yaml
+- $ref: wcsinfo.schema.yaml
 - $ref: bunit.schema.yaml
 - type: object
   properties:
```

Comment:
These two schemas have multiple SCI extensions, as indicated by the comments "An array of slits" and "An array of DrizProducts." If there is more than one SCI extension datamodels will only write to the first one. As I said in my original comment, I decided to only include wcsinfo when there is exactly one SCI extension. But if you feel differently, let me know.